tests/unit/string_lib_test.c

```diff
@@ -496,6 +496,72 @@ static void test_string_to_long_compatibility(void)
     // Old function (StringToLongUnsafe) does not allow trailing whitespace
 }
 
+#define ASSERT_INT_EQUAL_STRING_DECIMAL_TO_LONG(str, expected) \
+    assert_int_equal(StringDecimalToLong(str, &value_out), 0); \
+    assert_int_equal(value_out, expected)
```

Comment:
This macro should take value_out as a parameter. It is confusing that you have to declare a value_out in the same scope to use this macro.